docs/api/v3.rst

```diff
@@ -0,0 +1,727 @@
+    :>json string commit: A version control identifier for this build (eg. the commit hash)
+    :>json string builder: The hostname server that built the docs
+    :>json string cold_storage: Whether the build was removed from database and stored externally
+
+    :query boolean include_config: whether or not include the configs used for this build. Default is ``false``
+
+    :requestheader Authorization: optional token to authenticate.
+
+    :statuscode 200: Success
+    :statuscode 404: There is no ``Build`` with this ID
+
+
+.. http:get:: /api/v3/projects/(str:project_slug)/builds/latest/
+
+    Retrieve details for latest build on this project.
+
+    **Example request**:
+
+    .. sourcecode:: bash
+
+        $ curl https://readthedocs.org/api/v3/projects/pip/builds/latest/
+
+    **Example response**:
+
+    `See Build details <#build-details>`_
+
+    :requestheader Authorization: optional token to authenticate.
+
+
+Builds listing
+++++++++++++++
+
+.. http:get:: /api/v3/projects/(str:project_slug)/builds/
+
+    Retrieve list of all the builds on this project.
+
+    **Example request**:
+
+    .. sourcecode:: bash
+
+        $ curl https://readthedocs.org/api/v3/projects/pip/builds/
+
+    **Example response**:
+
+    .. sourcecode:: json
+
+        {
+            "count": 15,
+            "next": "/api/v3/projects/pip/builds?limit=10&offset=10",
+            "previous": null,
+            "results": ["BUILD"]
+        }
+
+    :query string commit: commit hash to filter the builds returned by commit
+    :query boolean running: whether or not to filter the builds returned by currently building
+
+    :requestheader Authorization: optional token to authenticate.
+
+
+Build triggering
+++++++++++++++++
+
+
+.. http:post:: /api/v3/projects/(string:project_slug)/builds/
+
+    Trigger a new build for this project.
+
+    **Example request**:
+
+    .. sourcecode:: json
+
+        {
+            "version": "latest",
+        }
+
+    **Example response**:
+
+    `See Build details <#build-details>`_
+
+    :requestheader Authorization: required token to authenticate.
+
+    :statuscode 201: Created sucessfully
+    :statuscode 400: Some field is invalid
+    :statuscode 401: Not valid permissions
+
+
+Build commands listing
+++++++++++++++++++++++
+
+.. http:get:: /api/v3/projects/(str:project_slug)/builds/(int:build_id)/commands/
+
+    Retrieve build command list of a single build.
+
+    **Example request**:
+
+    .. sourcecode:: bash
+
+        $ curl https://readthedocs.org/api/v3/projects/pip/builds/719263915/commands/
+
+    **Example response**:
+
+    .. sourcecode:: json
+
+        {
+            "count": 15,
+            "next": "/api/v3/projects/pip/builds/719263915/commands/?limit=10&offset=10",
+            "previous": null,
+            "results": ["BUILDCOMMAND"]
+        }
+
+    :requestheader Authorization: optional token to authenticate.
+
+
+Build command details
++++++++++++++++++++++
```

Comment:
Not sure if we really need this one, I can't imagine why will want to list only one command, this info should be retrieved in ``/api/v3/projects/(str:project_slug)/builds/(int:build_id)/commands/